### PR TITLE
11 user controller endpoint

### PR DESCRIPTION
**Pull request by ToastOfToastyness**

### CODE REVIEW

**Review by Wil-Lee — CHANGES_REQUESTED** [review]
Same here. Initialize all lokal mockUps in a constructor with zero parameters first. All mockUps will be Null with this implementation

**Review by Wil-Lee — APPROVED** [review]

File: src/main/java/com/peerrequest/app/api/UserController.java [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+
+    private Entry.EntrySelector entryId;
+
+    /**
+     * Method used to return all reviews of a specific user.
+     *
+     * @param userId the id of the user
+     * @return a list with all the reviews of the user
+     */
+    @GetMapping(value = "/users/{userId}/reviews", produces = "application/json")
```
Comment (bits0rcerer) [inline src/main/java/com/peerrequest/app/api/UserController.java:32]: Not defined in our [API](https://peerrequest.github.io/api_spec/#tag/users)

**Review by bits0rcerer — CHANGES_REQUESTED** [review]
Some endpoints under `/users` are missing.

**Review by Wil-Lee — APPROVED** [review]

**Comment by Wil-Lee** [comment]
I wasn't able to merge/rebase this branch with the master (mergeConflictLoop goes brrrrr), so I decided to create a new branch and start a pull request from there

**Pull request closed without merging**